src/test/java/pages/MainPage.java

```diff
@@ -0,0 +1,29 @@
+package pages;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class MainPage extends BasePage {
+
+    @FindBy(className = "auth-bar__item--text")
+    public static WebElement entrance_Button;
+
+    @FindBy(xpath = "//div[@class='auth-form__label-part auth-form__label-part_1']/following::div[@class='auth-form__label-title']")
```

Comment:
Is there any way to simplify this locator? can we use any xpath functions for that?

Reply:
simplified